pkg/operator/controller/dns/controller.go

```diff
@@ -735,14 +749,29 @@ func (r *reconciler) createDNSProvider(dnsConfig *configv1.DNS, platformStatus *
 		}
 	case configv1.PowerVSPlatformType:
 		// Power VS platform will use the ibm dns implementation
+
+		// Read custom service endpoints from the infra status and set when configured
+		// When none configured, the provider will use the default values
```

Comment:
Please update this comment as you did the corresponding comment in the `case configv1.IBMCloudPlatformType:` branch.